x-pack/test/security_solution_cypress/cypress/support/es_archiver.ts

```diff
@@ -19,16 +19,24 @@ export const esArchiver = (
   const log = new ToolingLog({ level: 'verbose', writeTo: process.stdout });
 
   const isServerless = config.env.IS_SERVERLESS;
+  const isCloudServerless = config.env.CLOUD_SERVERLESS;
+  const cloudUser = {
+    username: 'elastic',
+    password: config.env.ELASTICSEARCH_PASSWORD,
+  };
+  const serverlessUser = isCloudServerless ? cloudUser : systemIndicesSuperuser;
 
   const client = createEsClientForTesting({
     esUrl: Url.format(config.env.ELASTICSEARCH_URL),
     // Use system indices user so tests can write to system indices
-    authOverride: !isServerless ? systemIndicesSuperuser : undefined,
+    authOverride: !isServerless ? serverlessUser : undefined,
```

Comment:
This looks weird. why do we need to set the `serverlessUser` auth when we are **not** in serverless?
Is it possible to have _CLOUD_SERVERLESS=true_ and _IS_SERVERLESS=false_ at the same time?

Reply:
Synced with @semd and provided a new solution following his guidelines :)